docsrc/content/Differentiation.fsx

```diff
@@ -0,0 +1,39 @@
+(*** hide ***)
+// This block of code is omitted in the generated HTML documentation. Use 
+// it to define helpers that you do not want to show in the documentation.
+#I "../../bin/FSharp.Stats/netstandard2.0"
+(**
+Differentiation
+========================
```

Comment:
Thanks for adding documentation! The only thing i would miss here is a general introduction to numerical differentation, what h is etc.

Reply:
Thanks for all your feedback! I changed my code according to your suggestions!

I am looking forward to your reply 👍